reference/manifest/supporturl.md

```diff
@@ -1,27 +1,44 @@
 
 # SupportUrl element
-Specifies the URL of a page that provides support information for your add-in.
-
- **Add-in type:** Content, Task pane, Mail
 
+## Description
+Specifies the URL of a page that provides support information for your add-in.
 
-## Syntax:
-
+### Example
 
 ```XML
-<SupportUrl DefaultValue="string " />
-```
-
-
-## Can contain:
+<OfficeApp>
+...
+  <IconUrl DefaultValue="https://contoso.com/assets/icon-32.png" />
+  <HighResolutionIconUrl DefaultValue="https://contoso.com/assets/hi-res-icon.png"/>
+  
+  
+  <SupportUrl DefaultValue="https://contoso.com/support " />
+  
+  
+  <AppDomains>
+  ...
+  </AppDomains>
+...
+</OfficeApp>
 
-[Override](../../reference/manifest/override.md)
+```
 
 
 ## Attributes
 
-
-
 |**Attribute**|**Type**|**Required**|**Description**|
 |:-----|:-----|:-----|:-----|
 |DefaultValue|URL|required|Specifies the default value for this setting, expressed for the locale specified in the [DefaultLocale](../../reference/manifest/defaultlocale.md) element.|
+
+## Child elements
+
+|  Element | Required | Description  |
+|:-----|:-----|:-----|
+|  [Override](../../reference/manifest/override.md)   | No | Specifies the setting for additional locale urls |
+
+## Parent element
+[OfficeApp](../../reference/manifest/officeapp.md)
+
+## Additional Information
```

Comment:
Suggest we remove the Additional information heading, if we don't have any additional information to include here.